app/user/profile/LoanSlider.tsx

```diff
@@ -65,22 +66,24 @@ const LoanSlider = ({ works, loanData }: LoanSliderProps) => {
         <h2 className="text-typo-heading-4">
           E-materialer jeg har lånt ({loanData.loans?.length})
         </h2>
-        <div className="flex flex-row justify-end gap-x-4">
-          <Button
-            disabled={reachedStart}
-            variant="icon"
-            ariaLabel="Vis forrige værker"
-            onClick={() => onLeftClick()}>
-            <Icon className="h-[24px] w-[24px]" name="arrow-left" />
-          </Button>
-          <Button
-            disabled={reachedEnd}
-            variant="icon"
-            ariaLabel="Vis næste værker"
-            onClick={() => onRightClick()}>
-            <Icon className="h-[24px] w-[24px]" name="arrow-right" />
-          </Button>
-        </div>
+        {!!loanData.loans?.length && (
```

Comment:
You might as well just type:
```typescript
{loanData.loans?.length && (
Comment
```
because you just need to check the truthiness of the statement:
* if the `loans` property on `loanData.loans` is `undefined`it is falsy
* if the `loanData.loans.length`is 0 it is falsy

To me writing a double exclamation mark makes me wonder for a longer time what is actually going on.

Reply:
I had to reverse me changing this, because when the length of the array is 0 as a number, it renders the 0 in the frontend. So we do need the !! (boom! boom! you're a boolean) operator
![image](https://github.com/user-attachments/assets/7892fffb-86a5-4334-8bc1-8316d9d7a5a0)